docs/topics/db/models.txt

```diff
@@ -1417,6 +1417,79 @@ different database tables).
 Django will raise a :exc:`~django.core.exceptions.FieldError` if you override
 any model field in any ancestor model.
 
+
+.. _proxy-models-permissions-topic:
```

Comment:
I think this documentation might be more appropriate in topics/auth/default.html under the discussion of permission.